src/bindings/python/flux/job/stats.py

```diff
@@ -46,10 +46,10 @@ def __init__(self, handle, queue=None):
             "run",
             "cleanup",
             "inactive",
+            "successful",
             "failed",
             "timeout",
             "canceled",
-            "successful",
```

Comment:
Maybe the "reordering" done here should be moved to the first commit in this PR that also does reordering? Alternately, that commit could be moved last so that all the "reordering" is done in a single commit.

Reply:
ahhh that's a good idea, instead of having two commits do it